src/components/map/layers/NetworkLayers.tsx

```diff
@@ -100,7 +101,16 @@ export default class NetworkLayers extends Component<INetworkLayersProps> {
 
         return {
             // Layer name 'solmu' is directly mirrored from Jore through geoserver
-            solmu: (properties: any, zoom: any) => {
+            solmu: (properties: any, zoom: number) => {
+
+                const selectedTransitTypes = toJS(this.props.networkStore!.selectedTransitTypes);
+                if (properties.transittypes) {
+                    const transitTypes = TransitTypeHelper
+                        .convertTransitTypeCodesToTransitTypes(properties.transittypes.split(','));
```

Comment:
I suggested using array instead of string in your BE task, let's see what we agree on there before merging this parsing.

Reply:
Geoserver didn't seem to be able to handle an array from postgres. That was my intention at first, but I ended up needing to make it a strïng instead.